### PR TITLE
"Fix #2580: Reformatted the files

### LINKED ISSUE #2580: Reformatting Required for the following files
The following files require reformatting
 - layout/home_fragment.xml
 - layout-land/home_fragment.xml
 - layout-sw600dp-land/home_fragment.xml
 - layout-sw600dp-port/home_fragment.xml
 - layout/item_selection_interaction_items.xml
 - layout/lessons_chapter_view.xml
 - layout-land/lessons_chapter_view.xml
 - layout/multiple_choice_interaction_items.xml
 - layout/onboarding_slide.xml
 - layout-land/onboarding_slide.xml
 - layout-sw600dp-land/onboarding_slide.xml
 - layout-sw600dp-port/onboarding_slide.xml

Recommended to setup [Macros](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros) to solve this issue

References 
 - https://github.com/oppia/oppia-android/wiki/Coding-style-guide#code-formatting
 - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Explanation
Fixes #2580
I formatted the XML files using a macro for formatting files and optimizing imports

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by chrk2205 — CHANGES_REQUESTED** [review]
@Prince-kushwaha 
please revert the automated changes by android studio
as mentioned in the PR checklist 4.

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (chrk2205) [inline .idea/misc.xml]: revert these changes as they are changes made by android studio.
@Prince-kushwaha

Comment (Prince-kushwaha) [reply .idea/misc.xml]: ok it is add my mistake i fix this

File: .idea/misc.xml [hunk 2]
```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (chrk2205) [inline .idea/misc.xml]: you are supposed to revert everything i.e., auto generated by android studio 
simply everything under .idea folder
thanks@Prince-kushwaha

**Review by chrk2205 — CHANGES_REQUESTED** [review]
please remove everything that is auto generated by android studio thanks

**Comment by chrk2205** [comment]
This PR is not made form Develop branch so you can tick the 5th checklist also
@Prince-kushwaha 
if you remove the space in front of [ x] in checklist like this [x] it will show the tick instead

**Comment by Prince-kushwaha** [comment]
@chrk2205  I not known how  under .idea folder can you please tell me

**Comment by chrk2205** [comment]
> @chrk2205 I not known how under .idea folder can you please tell me

you just need to revert changes in misc.xml
@Prince-kushwaha

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.
